packages/cbioportal-frontend-commons/src/lib/getMobxPromiseGroupStatus.ts

```diff
@@ -1,13 +0,0 @@
-import * as _ from 'lodash';
-
-export function getMobxPromiseGroupStatus(
```

Comment:
renamed as `getRemoteDataGroupStatus` and moved under `cbioportal-utils`. `RemoteData` is an abstraction for `MobxPromise`.